### PR TITLE
Metrics App: Milestone final

### PR DESCRIPTION
**Pull request by Sevenpros**
In this pull request, I have built the metrics app including
- Add components
- Add actions
- Fetch API data
- Add data to store
- Display from the data store
- Add tests

### CODE REVIEW

File: src/components/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Countries from './Countries';
+
+const HomePage = () => (
+  <>
+    <div className="container">
+      <div className="headline">
+        <h2>CORONA VIRUS STATISTICS AROUND THE WORLD</h2>
+      </div>
+      <Countries />
```
Comment (t-yanick) [inline src/components/HomePage.js:9]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) for this project, " `When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.` ", please kindly add a filtering component to your home page(for a example a search). Checkout this [article](https://blog.logrocket.com/create-search-bar-react-from-scratch/) for inspiration.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @Sevenpros, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark: No `Linter` errors
:heavy_check_mark: Descriptive `PR` summary,
:heavy_check_mark: Great job with UI,
:heavy_check_mark: Clean/Organized code,
:heavy_check_mark: Good commit messages,

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments below and under the review._

- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) for this project: 

### Project documentation
Once you have finished the development of the project, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

- a description of the project.
- a demo of the project features.
- you should also highlight some interesting piece of code or something you built that you are very proud of.
For recording the video you can use tools like [Loom](https://www.loom.com/) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture. Please kindly add your project presentation video. 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/HomePage.js (hunk 1 above)
Comment (Sevenpros) [reply src/components/HomePage.js:9]: Hello @t-yanick  Hello thank you for the reviews you gave so much clarification that I was not aware of before, I have addressed all the issues mentioned in this review, but I would like to express my concern about recording the loom video, my computer has a temporary webcam problem so I could not record the video showing both screen and my face at the same time as it is in the project requirements, I hope you can bear with me while I am trying my best to fix the issue. thanks

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @Sevenpros , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**